keyboards/keebio/iris/keymaps/khang/keymap.c

```diff
@@ -0,0 +1,71 @@
+#include QMK_KEYBOARD_H
+
+enum keyboard_layers {
+    _BL = 0,
+    _FN,
+    _SL,
+    _BF,
+    _UP
+};
+
+#include "colors.c"
+#include "extrakeys.c"
```

Comment:
As a general rule, don't `#include` .c files, only `.h` files.

Reply:
Sure. Addressed in 6cc7d8d.